src/sbin/ping_hosts.sh

```diff
@@ -13,9 +13,11 @@
 #  Field 7:  Wan ping time threshold
 #  Field 8:  Address family
 
+IPSECHOSTS="/var/db/ipsecpinghosts"
+
 # Read in ipsec ping hosts and check the CARP status
-if [ -f /var/db/ipsecpinghosts ]; then
-	IPSECHOSTS="/var/db/ipsecpinghosts"
+# Check the file exists and is not empty
+if [ -f $IPSECHOSTS ] && [ -s $IPSECHOSTS ]; then
```

Comment:
I'd prefer moving this to enabled block below instead:

https://github.com/opnsense/core/blob/a7b1a7053356d82626f1848dde1b57b804286bf1/src/etc/inc/plugins.inc.d/ipsec.inc#L835

Reply:
Done, I tried to keep the script doing the same behavior than before. Still not knowing what consumes `currentipsecpinghosts`

Reply:
Okay, I was also thinking that the file put contents would create the file

Reply:
I still think something like https://github.com/opnsense/core/commit/893a7ab5612c25f901c375ead8bd6e627f981fe9 would fix your issue without modifying anything else

Reply:
Ohh, okay, got it